File: pkg/languages/java/detectors/testdata/string.java
public class Greet {
  const Greeting = "Hello World";

  public static void main(String[] args)
  {
    var s = Greeting + "!";
    s += "!!";

    String s2 = "hey ";
    s2 += args[0];
    s2 += " there";
  }
}

File: pkg/languages/python/detectors/testdata/string.py
class Greet:
    Greeting = "Hello World"

    def main(args):
        s = Greet.Greeting + "!"
        s += "!!"

        s2 = "hey "
        s2 += args[0]
        s2 += " there"

        s3 = f"foo '{s2}' bar"
